src/components/UI/Counter/Counter.tsx

```diff
@@ -44,7 +44,10 @@ const Counter = ({
       <Image
         src={theme === 'dark' ? minus : minusDark}
         alt="minus"
-        className={'cursor-pointer' + (count === 1 ? ' opacity-10 pointer-events-none' : '')}
+        className={
+          'cursor-pointer' +
+          (count === 1 ? ' pointer-events-none opacity-10' : '')
+        }
```

Comment:
This part was refactored in PR - 171.

Reply:
I realy don't remember when I made this changes in Counter.tsx)) Mabe the linter just format code. I count on this changes will be fixed when PR - 171 will be merged